doc/build-instructions/Build_Instructions_V8.md

```diff
@@ -377,7 +377,7 @@ export PATH="/cygdrive/c/Program Files/NASM:$PATH" (in Cygwin)
 Update your `INCLUDE` environment variable to provide a path to the Windows debugging tools with the following command:
 
 ```
-set INCLUDE=C:\Program Files\Debugging Tools for Windows (x64)\sdk\inc;%INCLUDE%
+export "INCLUDE=C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include;$INCLUDE"
```

Comment:
Ok, consistently using bash commands is probably better, but this should use single quotes, without the reference to the old value (which is not needed for building Java 8):
```
export INCLUDE='C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include'
```
(so `\` characters are taken literally, not as escapes)

Reply:
Updated.